### PR TITLE
DBM Doc Updates - SQLServer AlwaysOn & MySQL Aurora 8.0

### PR DESCRIPTION
**Pull request by caseycull**
### What does this PR do?
- Updates DBM MySQL docs to reflect support for MySQL Aurora 8.0.
- Updates DBM SQL Server setup docs to include AlwaysOn-specific agent configuration information.

### Motivation
Ensuring our DBM docs are up-to-date.

### Additional Notes

N/A

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/database_monitoring/setup_sql_server/azure.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ Do the following steps to enable Database Monitoring with your database:
 2. [Install the Agent](#install-the-agent)
 3. [Install the Azure integration](#install-the-azure-integration)
 
+**For AlwaysOn users**, the agent must be installed on a separate server and connected to the cluster through the listener endpoint, as information about Availability Group (AG) secondary replicas is collected from the primary replica. Additionally, installing the agent in this way helps keep it up and running in the event of a failover.
```
Comment (hestonhoffman) [inline content/en/database_monitoring/setup_sql_server/azure.md:28]:
```suggestion
**For AlwaysOn users**, the Agent must be installed on a separate server and connected to the cluster through the listener endpoint, as information about Availability Group (AG) secondary replicas is collected from the primary replica. Additionally, installing the Agent in this way helps keep it up and running in the event of a failover.
```

File: content/en/database_monitoring/setup_sql_server/gcsql.md [hunk 2]
```diff
@@ -22,6 +22,8 @@ Complete the following steps to enable Database Monitoring with your database:
 2. [Install the Agent](#install-the-agent)
 3. [Install the Cloud SQL integration](#install-the-cloud-sql-integration)
 
+**For AlwaysOn users**, the agent must be installed on a separate server and connected to the cluster through the listener endpoint, as information about Availability Group (AG) secondary replicas is collected from the primary replica. Additionally, installing the agent in this way helps keep it up and running in the event of a failover.
```
Comment (hestonhoffman) [inline content/en/database_monitoring/setup_sql_server/gcsql.md:25]:
```suggestion
**For AlwaysOn users**, the Agent must be installed on a separate server and connected to the cluster through the listener endpoint, as information about Availability Group (AG) secondary replicas is collected from the primary replica. Additionally, installing the Agent in this way helps keep it up and running in the event of a failover.
```

File: content/en/database_monitoring/setup_sql_server/rds.md [hunk 3]
```diff
@@ -25,6 +25,8 @@ Do the following steps to enable Database Monitoring with your database:
 2. [Install the Agent](#install-the-agent)
 3. [Install the RDS integration](#install-the-rds-integration)
 
+**For AlwaysOn users**, the agent must be installed on a separate server and connected to the cluster through the listener endpoint, as information about Availability Group (AG) secondary replicas is collected from the primary replica. Additionally, installing the agent in this way helps keep it up and running in the event of a failover.
```
Comment (hestonhoffman) [inline content/en/database_monitoring/setup_sql_server/rds.md:28]:
```suggestion
**For AlwaysOn users**, the Agent must be installed on a separate server and connected to the cluster through the listener endpoint, as information about Availability Group (AG) secondary replicas is collected from the primary replica. Additionally, installing the Agent in this way helps keep it up and running in the event of a failover.
```

File: content/en/database_monitoring/setup_sql_server/selfhosted.md [hunk 4]
```diff
@@ -69,6 +69,8 @@ CREATE USER datadog FOR LOGIN datadog;
 
 It's recommended to install the agent directly on the SQL Server host as that enables the agent to collect a variety of system telemetry (CPU, memory, disk, network) in addition to SQL Server specific telemetry.
 
+**For AlwaysOn users**, the agent must be installed on a separate server and connected to the cluster through the listener endpoint, as information about Availability Group (AG) secondary replicas is collected from the primary replica. Additionally, installing the agent in this way helps keep it up and running in the event of a failover.
```
Comment (hestonhoffman) [inline content/en/database_monitoring/setup_sql_server/selfhosted.md:72]:
```suggestion
**For AlwaysOn users**, the Agent must be installed on a separate server and connected to the cluster through the listener endpoint, as information about Availability Group (AG) secondary replicas is collected from the primary replica. Additionally, installing the Agent in this way helps keep it up and running in the event of a failover.
```

**Review by hestonhoffman — APPROVED** [review]
Some small edits

**Comment by caseycull** [comment]
@hestonhoffman thanks for the review!

**Comment by caseycull** [comment]
@alexbarksdale from my team will take a look as well on Monday

**Comment by alexbarksdale** [comment]
The added notes about AO look good, but I do have one question. There are necessary config options that need to be enabled to use AlwaysOn, do you think this should be included here?

**Review by alexbarksdale — APPROVED** [review]

**Review by cswatt — APPROVED** [review]

**Pull request merged**